ruby/rails/security/audit/rails-check-page-caching-cve.yaml

```diff
@@ -0,0 +1,52 @@
+rules:
```

Comment:
For this rule, we also want a ruby rule that catches instances of code like 
```
class BooksController < ApplicationController
caches_page :show
end
```
(where show is vulnerable).
Good work on catching the page-caching gems though 👍